control/trajectory_follower/include/trajectory_follower/mpc_lateral_controller.hpp

```diff
@@ -187,7 +191,7 @@ class TRAJECTORY_FOLLOWER_PUBLIC MpcLateralController : public LateralController
   /**
    * @brief check ego car is in stopped state
    */
-  bool isStoppedState() const;
+  bool isStoppedState();
```

Comment:
No need to remove const?

Reply:
`isSteerConverged` is used in this funcion.
and `isSteerConverged` adds a marker to a member variable. 
so removing const is needed if making `m_debug_marker_array` a member.